joplin/js/CreateContentModal/ChooseTitleStep.js

```diff
@@ -83,8 +83,7 @@ const ChooseTitleStep = ({
             Use words you think residents may search to find the {pageType}.
           </li>
           <li>
-            You don’t need to worry about including your department’s name in
-            the title.
+            You don’t need to include your department’s name in the title.
```

Comment:
this is the microcopy change for issue #3444 (replace joplin microcopy for content types)

Reply:
I spun this out into its own PR and branch now